src/api_emulator/redfish/event_service_api.py

```diff
@@ -214,10 +214,12 @@ def post(self):
         ident = '%d' % id
         
         destination = raw_dict['Destination']
-        event_types = raw_dict['EventTypes']
-        for evType in event_types:
-            if evType not in e_config['EventTypesForSubscription']:
-                return 'Invalid EventType %s' % evType, 400
+        event_types = []
```

Comment:
This change isn't quite right. A range around this area needs to be made: https://github.com/Cray-HPE/csm-redfish-interface-emulator/blob/master/src/api_emulator/redfish/event_service_api.py#L114

Ideally somehow the loader should be able to flip a flag/switch if Gigabyte mockup is detected to allow the Empty EventType.